config/prow/README.md

```diff
@@ -3,10 +3,8 @@
 This directory contains the config for our
 [Prow](https://github.com/kubernetes/test-infra/tree/master/prow) instance.
 
-- `Makefile` Commands to interact with the Prow instance regarding configs and
-  updates. Run `make help` for assistance.
-- `cluster/*.yaml` Deployments of the Prow cluster.
-- `core/*.yaml` Generated core configuration for Prow.
+- `../Makefile` Commands to interact with the Prow instance regarding configs and updates. Run `make help` for assistance.
+- `build-cluster/cluster/*.yaml` Deployments of the Prow cluster.
```

Comment:
The layout in https://github.com/GoogleCloudPlatform/oss-test-infra/tree/master/prow/knative/cluster actually makes more sense so I would recommend directly porting them over here, and merge the existing config files in knative/test-infra into that layout. 

Let me know if you have questions regarding this layout, and I can walk you through it via video chat.